SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -235,11 +235,13 @@ SCDocHTMLRenderer {
 		<< "<link rel='stylesheet' href='" << baseDir << "/editor.css' type='text/css' />\n"
 		<< "<link rel='stylesheet' href='" << baseDir << "/frontend.css' type='text/css' />\n"
 		<< "<link rel='stylesheet' href='" << baseDir << "/custom.css' type='text/css' />\n"
+    	<< "<link rel='stylesheet' href='" << baseDir << "/lib/katex/katex.min.css' />\n"
```

Comment:
The indentation is off here, is it a tabs Vs spaces thing?

Reply:
well spotted! vscode converts indentations form tab to 4 spaces which happened here - have changed my config and will try to reduce noise by rebasing the commit.